src/lib/patterns.js

```diff
@@ -24,10 +24,19 @@ const glob = require('fast-glob');
 const fs = require('fs');
 const path = require('path');
 const matter = require('gray-matter');
+const {forceForwardSlash} = require('./utils/path');
 
 const stripDot = /^\./;
-const basePath = path.join(__dirname, '../../src/site/content/en/patterns');
-const files = glob.sync(path.join(basePath, '**', 'index.md'));
+const contentRoot = path.join(__dirname, '../../src/site/content/en');
+const basePath = path.join(contentRoot, 'patterns');
+const globBasePath = path.join(
+  contentRoot,
+  '{patterns,handbook/content-types/example-pattern}',
+);
+
+const files = glob.sync(
+  forceForwardSlash(path.join(globBasePath, '**', 'index.md')),
+);
```

Comment:
I find it hard to grasp what's happening here. I think there are two things specifically that I stumble over: 

`forceForwardSlash` is needed for fast-glob to correctly handle Windows paths right? I wonder if that's still needed if path formats wouldn't be mixed, but instead all built by `path.join`.

Would that work with the following approach?

```js
const contentRoot = path.join(__dirname, '..', '..', 'src', 'site', 'content', 'en');
const basePath = path.join(contentRoot, 'patterns');

const patternPaths = [
  path.join(contentRoot, 'patterns', '**', 'index.md'),
  path.join(contentRoot, 'handbook', 'content-types', 'example-pattern', '**', 'index.md'),
]

const patterns = glob.sync(patternPaths);
```

I didn't test this, but the idea would be to let path.join generate full paths, instead of joining partials - if that makes sense...?